src/main/java/org/assertj/core/api/AbstractClassAssert.java

```diff
@@ -284,4 +329,111 @@ public SELF hasDeclaredFields(String... fields) {
     classes.assertHasDeclaredFields(info, actual, fields);
     return myself;
   }
+
+  /**
+   * Verifies that the actual {@code Class} has the {@code methods}.
```

Comment:
reword: `has the given methods.` I use `{@code` for something that denotes a java element like {@code true} or {@code equals}